chore: Update DBUS_PATTERN and DBUS_WATCH_EXPRESSION description (#2613)
mendersoftware/mender-docs: 10.Add-ons/20.Monitor/40.Configuration/docs.md

### DBUS_PATTERN

The grep pattern to match on. An empty value will match on all notifications.

> At leaset DBUS_PATTERN or DBUS_WATCH_EXPRESSION is required to be set to avoid messaging flood.

Reviewer: `At leaset` -> `At least`
Or rephrase it to something like:
```suggestion
To avoid message flooding, it is required to set either `DBUS_PATTERN` or `DBUS_WATCH_EXPRESSION`.
```
Same below

Author: Thanks for the suggestion, I have updated the text accordingly
